src/TimeSteppers/clock.jl

```diff
@@ -42,9 +43,21 @@ Base.show(io::IO, c::Clock{T}) where T =
                     ", iteration = ", c.iteration,
                         ", stage = ", c.stage)
 
+tick_time(clock, Δt) = clock.time + Δt
+tick_time(clock::Clock{<:AbstractTime}, Δt) = clock.time + Nanosecond(round(Int, 1e9 * Δt))

```

Comment:
How about `next_time` so we don't get confused with the mutating function `tick_time!`? Unless I missed something.

Reply:
Yup that makes sense!